.flake8

```diff
@@ -1,5 +1,33 @@
 [flake8]
-ignore = E203, E266, E501, W503, F403, F401
+ignore =
+    E203
+    E266
+    E501
+    W503
+    F403
+    F401
+    D401  # Imperative mood -- doesn't work for cached-property.
+    N802  # TODO: remove this (functino name should be lower-case)
+
 max-line-length = 88
 max-complexity = 18
-select = B,C,E,F,W,T4,B9
```

Comment:
genuine question here, what does the removing select do? Does it just run everything?

Reply:
Yeah. It runs everything. Which is why I added very specific ignores :-) The plan is NOT to ignore anything except very specific things eventually. I've always just copy/pasted this select argument -- but you'll see it doesn't select `D` which is documentation...